cob_bringup/drivers/relayboard.launch

```diff
@@ -6,7 +6,11 @@
 	<arg name="sim" default="false" />
 
 	<!-- start simulated relayboard node -->
-	<node if="$(arg sim)" ns="relayboard" pkg="cob_relayboard" type="relayboard_sim.py" name="relayboard_node" respawn="false" output="screen"/>
+	<node if="$(arg sim)" ns="relayboard" pkg="cob_relayboard" type="relayboard_sim.py" name="relayboard_node" respawn="false" output="screen">
+		<remap from="/emergency_stop_state" to="emergency_stop_state"/>
+		<remap from="/power_board/state" to="power_board/state"/>
+		<remap from="/power_board/voltage" to="power_board/voltage"/>
+	</node>
```

Comment:
this better should be handled in changing the NodeHandle and the topics in https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_relayboard/ros/src/cob_relayboard_node.cpp#L96-L100

There, it should rather look something like this:
```
n = ros::NodeHandle();
n_priv = ros::NodeHandle("~");

topicPub_isEmergencyStop = n.advertise<cob_msgs::EmergencyStopState>("emergency_stop_state", 1);
topicPub_PowerBoardState = n.advertise<cob_msgs::PowerBoardState>("power_board/state", 1);
topicPub_Voltage = n.advertise<std_msgs::Float64>("power_board/voltage", 10);
```
In all occurences of n within the relayboard node, where n is used with either `hasParam()`, `getParam` or `param()`, it should be replaced by n_priv!